### PR TITLE
CLDR-15017 Check for back-and-forth flip-flop values

### PR DESCRIPTION
**Pull request by btangmu**
-New FindFlipFlops.java, partly based on GenerateBirth.java

-List all paths in all locales where value changed from A to B then back to A

CLDR-15017

- [ ] This PR completes the ticket.

### CODE REVIEW

**Comment by btangmu** [comment]
@macchiati when I run this, 12,590 flip-flops are found; the first and last items are:
🏓🏓 1 //ldml/numbers/currencies/currency[@type="PHP"]/displayName[@count="one"] en
Philippine peso
Philippine peso
Philippine piso
Philippine piso
Philippine piso
Philippine piso
Philippine piso
Philippine piso
Philippine piso
Philippine piso
Philippine piso
Philippine piso
Philippine piso
Philippine piso
Philippine piso
Philippine peso
Philippine peso
Philippine peso
Philippine peso
Philippine peso
Philippine peso
Philippine peso
Philippine peso
Philippine peso
Philippine peso
Philippine peso
Philippine peso
Philippine peso
Philippine peso
Philippine peso
Philippine peso
...
...
🏓🏓 12590 //ldml/dates/fields/field[@type="day"]/displayName zu
Usuku
Usuku
Usuku
Usuku
Usuku
Usuku
Usuku
Usuku
Usuku
Usuku
Usuku
Usuku
Usuku
Usuku
Usuku
Usuku
Usuku
Usuku
Usuku
usuku
usuku
Usuku
Usuku
Usuku

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/FindFlipFlops.java [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+                continue; // skip region locales
+            }
+            System.out.println(fileName);
+            findFlops(fileName);
+        }
+        System.out.println("Total " + flipFlopCount + " flip-flops");
+    }
+
+    private static void confirmVersionArchiveIsPresent() {
+        try {
```
Comment (btangmu) [inline tools/cldr-code/src/main/java/org/unicode/cldr/tool/FindFlipFlops.java:64]: some of these methods are based on code in GenerateBirth.java -- if this pr is accepted, some code should go in a shared utility class, in a subsquent pr

**Review by macchiati — CHANGES_REQUESTED** [review]
I think this is almost there, but the information needs to be in a form that makes it easy for use to see patterns.

That is, we need to see not just a raw listing of the same string over and over again (which from the description is what is produced), but rather when each change was made. Eg,
``
#Version[TAB]From[TAB]⇒[TAB]To
21.0[TAB]«abc»[TAB]⇒[TAB]«adc»
23.0[TAB]«adc»[TAB]⇒[TAB]«abc»
``

And (it is hard to tell whether you are doing this from the listing) a line should only be listed if 
  * its From value equals some other lines To value, or 
  * its To value equals some other line's From value.

**Comment by btangmu** [comment]
See https://github.com/unicode-org/cldr/pull/1991

**Pull request closed without merging**